src/app/guards/configuration.guard.ts

```diff
@@ -8,7 +8,7 @@ import { Configuration } from '../models/configuration';
 @Injectable({
     providedIn: 'root',
 })
-export class ConfigurationGuard implements CanActivate {
```

Comment:
This isn't a proper migration to the new style of defining guards.
I would prefer to migrate to the new style immediately.

See the links for the new style:
* https://angular.io/guide/deprecations#router-class-and-injectiontoken-guards-and-resolvers
* https://angular.io/guide/router#preventing-unauthorized-access

You can also use `ng g` to see how it is done.

Reply:
done